### PR TITLE
mssql-store-adapter

### PR DESCRIPTION
**Pull request by zacmos**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,12 +30,14 @@
...
   "dependencies": {
+    "@types/mssql": "^6.0.7",
+    "@types/node-cron": "^2.0.3",
     "async-mutex": "^0.2.4",
     "express": "^4.17.1",
     "is-valid-path": "^0.1.1",
+    "mssql": "^6.3.1",
     "nanoid": "^3.1.16",
     "node-cron": "^2.0.3",
-    "@types/node-cron": "^2.0.3",
```
Comment (duckranger) [inline package.json:38]: Why was this removed?

**Review by duckranger — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (zacmos) [reply package.json:38]: It's not removed but moved up in the file..

**Review by duckranger — APPROVED** [review]
